xml/cl.xml

```diff
@@ -1719,8 +1719,12 @@ server's OpenCL/api-docs repository.
             <unused start="0x4213" end="0x421F"/>
     </enums>
 
-    <enums start="0x4220" end="0x4FFF" name="enums.4220" comment="Reserved for vendor extensions. Allocate in groups of 16.">
-            <unused start="0x420F" end="0x4FFF"/>
+    <enums start="0x4220" end="0x4230" name="enums.4220" vendor="IMG">
```

Comment:
This reserves 17 values but reserving is done in blocks of 16.   If you actually need 17 values then reserve a block of size 32.

Reply:
Woops, thats embarrassing. Should be fixed now.

Reply:
Thanks!